cfa.gemspec

```diff
@@ -1,11 +1,12 @@
 Gem::Specification.new do |s|
-  s.name        = "config_files_api"
+  s.name        = "cfa"
   s.version     = "0.1.0"
   s.platform    = Gem::Platform::RUBY
   s.authors     = ["Josef Reidinger"]
   s.email       = ["jreidinger@suse.cz"]
   s.homepage    = "http://github.com/config-files-api/config_files_api"
-  s.summary     = "Easy way to create model on top of configuration file"
+  s.license     = "LGPLv3"
+  s.summary     = "CFA (Config Files API) provides easy way to create model on top of configuration file"
```

Comment:
"provides an easy way"
Choose one
"create models on top of configuration files"
"create a model on top of a configuration file"